web/src/screens/library/LibraryPageScreen/blocks/LibraryPageTableBlock/PropertyListMenu/PropertyListMenu.tsx

```
@@ -0,0 +1,124 @@
+      onSelect={handleSelect}
+    >
+      <Menu.Trigger asChild>{children}</Menu.Trigger>
+
+      <Portal>
+        <Menu.Positioner>
+          <Menu.Content minW="36">
+            <Menu.ItemGroup pl="2" py="1">
+              <Menu.ItemGroupLabel>Properties</Menu.ItemGroupLabel>
+
+              {properties.map((property) => (
+                <Menu.Item value={property.fid}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)